common_benchmark_suite/openxla/benchmark/comparative_suite/jax/scripts/generate_model_artifacts.py

```diff
@@ -78,6 +79,16 @@ def _generate_artifacts(model: def_types.Model, save_dir: pathlib.Path,
 
     print(f"Completed generating artifacts {model.name}\n")
 
+    if auto_upload:
+      subprocess.run(
+          f"gsutil cp -r {model_dir} {GCS_UPLOAD_DIR}/{save_dir.name}/{model_dir.name}",
```

Comment:
I think `gcloud storage cp` is a newer version of the gcloud cli and we use it elsewhere. Maybe change to it for consistency?

Reply:
Actually, I get permissions issues if I use `gcloud storage cp` instead of `gsutil cp`:

```
ERROR: (gcloud.storage.cp) HTTPError 403: ... does not have serviceusage.services.use access to the Google Cloud project. Permission 'serviceusage.services.use' denied on resource (or it may not exist).
```

Reply:
Could you test this command locally?

```
touch test.txt && gcloud storage cp test.txt gs://iree-model-artifacts/tensorflow/tf_models_2.14.0rc0_1692307801/
```

Maybe re-initialize the session with `gcloud init` will also help.

I tested this change with `gcloud storage cp` and it seems to work for me. I think eventually we want to run these tools in CI, and `gsutil` might not be available on runners.

Reply:
It still doesn't work for me but we can figure that out separately. I changed it to use `gcloud storage`.